python/docs/examples/bluesky.py

```diff
@@ -26,7 +26,6 @@ async def main():
     # The firehose doesn't (currently) require authentication.
     at_client = AsyncFirehoseSubscribeReposClient()
 
-    # [start_setup]
```

Comment:
It may be worth keeping these comments, and hoping to switch back to them later.

1. They won't show up if we choose lines in between the tags.
2. It lets us know what lines are used or not though while we look at the script.